docs/auth_rules.md

```diff
@@ -0,0 +1,33 @@
+# Current implemented rules in auth_map
+| Transaction type | Field | Previous value | New value | Who can| Description |
+|------------------|-------|----------------|-----------|--------|-------------|
+| NYM              |`role` |`<empty>`       | TRUSTEE   | TRUSTEE|Adding new TRUSTEE|
+| NYM              |`role` |`<empty>`       | STEWARD   | TRUSTEE|Adding new STEWARD|
+| NYM              |`role` |`<empty>`       | TRUST_ANCHOR| TRUSTEE, STEWARD|Adding new TRUST_ANCHOR|
+| NYM              |`role` |`<empty>`       |`<empty>`  | TRUSTEE, STEWARD, TRUST_ANCHOR| Adding new nym|
+| NYM              |`role` | TRUSTEE        |`<empty>`  | TRUSTEE | Change role from TRUSTEE to None|
+| NYM              |`role` | STEWARD        |`<empty>`  | TRUSTEE | Change role from STEWARD to None|
+| NYM              |`role` | TRUST_ANCHOR   |`<empty>`  | TRUSTEE | Change role from TRUST_ANCHOR to None|
+| NYM              |`verkey`|`*`|`*`| Owner of this nym | Any operations with verkey field|
+| SCHEMA           |`*`|`*`|`*`| TRUSTEE, STEWARD, TRUST_ANCHOR | Any operations with SCHEMA transactions|
+| CLAIM_DEF        |`*`|`*`|`*`| Owner of this claim_def txn| Any operations with CLAIM_DEF transaction|
```

Comment:
Need to split to `Adding new Claim Def` (TRUST_ANCHOR only?) and `Editing existing ClaimDef` (Owner only?)